<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alternativeInput</class>
 <widget class="QDialog" name="alternativeInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>509</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1666666</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>с 3 урока: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>на 6 урок: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QSpinBox" name="min_les_8">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>с 2 урока: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="15">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Часы</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>23</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_2">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_3">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_4">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_5">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_6">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_7">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_8">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_9">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_10">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_11">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_12">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_13">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="hours_les_14">
         <property name="maximum">
          <number>23</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>на 3 урок: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>на 2 урок: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>с 6 урока: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="min_les_1">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="min_les_2">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Звонок</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QSpinBox" name="min_les_9">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="min_les_4">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="min_les_6">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>на 7 урок: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>с 1 урока: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>с 4 урока: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QSpinBox" name="min_les_7">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="min_les_5">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Минуты</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>с 7 урока: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>на 1 урок: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>на 4 урок: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="min_les_3">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>на 5 урок: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>c 5 урока: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QSpinBox" name="min_les_10">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QSpinBox" name="min_les_11">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QSpinBox" name="min_les_12">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QSpinBox" name="min_les_13">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QSpinBox" name="min_les_14">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>alternativeInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>alternativeInput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
